src/__test__/redux/reducers/componentConfig/loadPlotData.test.js

```diff
@@ -0,0 +1,63 @@
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
+import { act } from 'react-dom/test-utils';
+
+import { makeStore } from 'redux/store';
+
+import plotNames from 'utils/plots/plotNames';
+import fake from '__test__/test-utils/constants';
+
+import { loadPlotData, loadPlotConfig } from 'redux/actions/componentConfig';
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+import { fetchWork } from 'utils/work/fetchWork';
+
+enableFetchMocks();
+
+// Return 404 to use default plot config
+fetchMock.mockResponse(() => ({
+  status: 404,
+  body: 'Not Found',
+}));
+
+jest.mock('utils/pushNotificationMessage');
+jest.mock('utils/work/generatePlotWorkBody');
+jest.mock('utils/work/fetchWork', () => ({
+  fetchWork: jest.fn(() => Promise.resolve(
+    ['work-result-1', 'work-result-2'],
+  )),
+}));
+
+const experimentId = fake.EXPERIMENT_ID;
+let storeState = null;
+
+const plotUuid = 'DotPlotMain';
+const plotType = plotNames.plotType.DOT_PLOT;
+
+describe('plotDataLoaded', () => {
+  beforeEach(() => {
+    storeState = makeStore();
+
+    storeState.dispatch(loadPlotConfig(plotUuid, plotType));
+  });
+
+  it('Loads plot data into the component properly', async () => {
+    await act((async () => {
+      await storeState.dispatch(loadPlotData(experimentId, plotUuid, plotType));
+    }));
+
+    const { plotData } = storeState.getState().componentConfig[plotUuid];
+
+    expect(plotData.length).toBeGreaterThan(0);
+  });
+
+  it('Throws an error if there is an error fetching plot data', async () => {
+    fetchWork.mockImplementation(() => {
+      throw new Error('Error fetching work');
+    });
+
+    await act((async () => {
+      await storeState.dispatch(loadPlotData(experimentId, plotUuid, plotType));
+    }));
+
+    expect(pushNotificationMessage).toHaveBeenCalled();
```

Comment:
1. add how many times do you expect the function to be called
2. is checking if the function has been called enough to test that `throws an error if there is an error fetching plot data`?

Reply:
Whoops, it should display notification, not throw an error.

Reply:
I've also modified `toHaveBeenCalled()` into `toHaveBeenCalledTimes(1)`